src/types.ts

```diff
@@ -41,6 +41,7 @@ export interface ConnectorConfig
 }
 
 export interface VizConfig extends CommonConfig, VizConfigHasDefaults {
+  codelab: boolean;
```

Comment:
This should probably be in the `VizConfigHasDefaults` type (and the default would be false)